client/components/LearnerViewDirective.js

```diff
@@ -477,10 +477,22 @@ tie.directive('learnerView', [function() {
             feedbackDiv.scrollTop = feedbackDiv.scrollHeight +
               additionalHeightForLoadingIndicator;
             $timeout(function() {
-              SolutionHandlerService.processSolutionAsync(
-                tasks[currentTaskIndex], code,
+              var index = 0;
```

Comment:
@seanlip Please view our design doc. We are ready to discuss the impl once you've finished reading.